provisioner/terraform/resources.go

```diff
@@ -853,6 +853,17 @@ func ConvertState(ctx context.Context, modules []*tfjson.StateModule, rawGraph s
 			Name:       preset.Name,
 			Parameters: presetParameters,
 		}
+
+		if len(preset.Prebuild) > 1 {
```

Comment:
There was a todo here about whether or not to allow more than one prebuild definition in a preset. I took the initiative to say no, for now.

Commenting here so that reviewers can challenge the decision if they'd like.
Its something we can change though, so I'd prefer to keep this here for now in the interest of actually shipping it.